### ISSUE TITLE
Possible mistranslation in Chinese (simplified) zh-CN version in chapter2/2.mdx

### ISSUE THREAD

**Issue by MaxLong2001** [huggingface/course#492]
[en chapter2/2.mdx](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/2.mdx#a-high-dimensional-vectora-high-dimensional-vector)
[zh-CN chapter2/2.mdx](https://github.com/huggingface/course/blob/main/chapters/zh-CN/chapter2/2.mdx#%E9%AB%98%E7%BB%B4%E5%90%91%E9%87%8F)
In the 2nd section of chapter2 (links above), when it comes to the _last paragraph_ of **A high-dimensional vector** (**高维向量** in Chinese), the original content `outputs["last_hidden_state"]` should be _referring to a patch of code_. But the Chinese version made a translation and writes `输出["last_hidden_state"]` , which I believe is not appropriate. It should be kept untranslated just like the following `outputs[0]`.

**Comment by tyisme614** [comment]
correct,  'outputs' here refers to a variable which should not be translated.
